app/webpacker/packs/application.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable import/first */
 require.context("govuk-frontend/govuk/assets");
 
+import "core-js";
```

Comment:
we should probably only include what we need (i think there's a babel plugin or webpack config that can do it for us though)

Reply:
I was trying to use this https://babeljs.io/docs/en/babel-preset-env#usebuiltins,  but perhaps I used it wrong?

Reply:
Oh awesome, they've changed how that works! Looks good to me, then.